controllers/argocd/configmap.go

```diff
@@ -539,6 +557,50 @@ func (r *ReconcileArgoCD) reconcileRBACConfigMap(cm *corev1.ConfigMap, cr *argop
 	return nil // ConfigMap exists and nothing to do, move along...
 }
 
+// validateOwnerReferences checks if OwnerReferences is changed
+func validateOwnerReferences(cr *argoproj.ArgoCD, cm *corev1.ConfigMap, scheme *runtime.Scheme) (bool, error) {
+	changed := false
+
+	if cm.OwnerReferences != nil {
+		ref := cm.OwnerReferences[0]
```

Comment:
Thanks for the revivew @svghadi. It is done this way because we don't support/recommend multiple Argo CD instances in one namespace, so there should be a single OwnerReference in ConfigMap.